tests/h/exceptions_test.py

```diff
@@ -49,12 +49,7 @@ def test_it_sets_type_and_message(self):
     def test_it_sets_default_response_code(self):
         exc = exceptions.OAuthTokenError("boom", "mytype")
 
-        assert exc.status_code == 400
-
-    def test_it_sets_custom_response_code(self):
-        exc = exceptions.OAuthTokenError("boom", "mytype", 500)
-
-        assert exc.status_code == 500
+        assert exc.status_code == 401
```

Comment:
The default `OAuthTokenError.status_code` changed from `400` to `401`. In practice the default was never used and `401` was always passed in as a custom value. Now the default is `401` and a custom value never needs to be passed.